solutions/banking/main.tf

```diff
@@ -246,6 +246,26 @@ resource "ibm_cd_tekton_pipeline_property" "watsonx_assistant_id_pipeline_proper
   value       = var.watson_assistant_instance_id
 }
 
+# Update CI pipeline with Assistant instance region
+resource "ibm_cd_tekton_pipeline_property" "watsonx_assistant_region_pipeline_property_ci" {
+  depends_on  = [local.cd_instance]
+  provider    = ibm.ibm_resources
+  name        = "watson_assistant_region"
```

Comment:
Can you change this to watsonx_assistant_region? Only the name is important as we have the app scripts looking for that value, otherwise we'd have to change those.